ocs_ci/helpers/storageclass_helpers.py

```diff
@@ -0,0 +1,54 @@
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+
+
+def storageclass_name(interface, external_mode=False):
```

Comment:
suggest renaming to `get_default_storage_class_name`

Reply:
Done